src/sentry/api/serializers/models/event.py

```diff
@@ -99,6 +99,11 @@ def serialize(self, obj, attrs, user):
             except TypeError:
                 received = None
 
+        event_type = obj.data.get('type', 'default')
+        metadata = obj.data.get('metadata') or {
+            'title': obj.message_short,
+        }
```

Comment:
we are worried (need to coerce it both on ingest and deserialize, and since we only use it in API stuff, its ok to just do it here)